### PR TITLE
DFC-45 | GA4 | Sandbox | Create form - select component on test page

### PR DESCRIPTION
**Pull request by SophiaDWokoma**
### Description ###

-  Creates chooseLocation.njk file and defines a route for it in the app.js file
- Introduces a select component to chooseLocation file .
-  Implements error handling for the select form.
-  Redirects to the home page when the user enters text and clicks continue since the manage form submission page is not yet created.
- Modified the generic validateForm function so that it could still be used for select form as form has a default value selected on load.

### Tickets ###
(DFC-45)[https://govukverify.atlassian.net/browse/DFC-45]

### Steps to Reproduce ###

1. Navigate to /choose-location page
2. Test the error handling of the form.
3. Verify the redirection functionality to the home page when item is selected.

### Additional Information ###

Saving data from form entry will be added later/ another ticket

### CODE REVIEW

**Review by di-fabs — CHANGES_REQUESTED** [review]

File: src/validateForm.js
Comment (SophiaDWokoma) [reply src/validateForm.js]: ok I have changed the function to check the length and removed the errorValue parameter

File: src/validateForm.js
Comment (di-fabs) [inline src/validateForm.js:2]: Best condition is `(!formData || !formData.length)`

Explaination: false = 0, true = anything else, so you can just check if formData.length === false (0) => !formData.length

**Comment by Joanneyoung01** [comment]
Looks good to me, will approve once the merge conflict is resolved :D

**Review by di-fabs — APPROVED** [review]

**Comment by SophiaDWokoma** [comment]
> Looks good to me, will approve once the merge conflict is resolved :D

have resolved

**Review by Joanneyoung01 — APPROVED** [review]

**Pull request merged**
